lang/main-sv.json

```diff
@@ -1159,7 +1356,7 @@
         "go": "KÖR",
         "goSmall": "BÖRJA",
         "headerSubtitle": "Säkra möten med hög kvalitet",
-        "headerTitle": "Jitsi Meet",
+        "headerTitle": "Boverket videomöten",
```

Comment:
Is this correct?

Reply:
@damencho no. I had a clean one and a customer one and commited the wrong file. Ill fix that.